### PR TITLE
added new topic oncourseCI

### PR DESCRIPTION
**Pull request by AshishVarshneyy**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Concourse CI/CD is an open-source platform that runs on two main Docker containers. Once Concourse is installed, all you have to do is to use the docker-compose up command to bring up the Concourse server. Concourse uses Postgres as its database.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [x] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/concourseCli/index.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+display_name: concourseCli
```
Comment (deniseyu) [inline topics/concourseCli/index.md:2]: The name of the project is just "Concourse CI". It does have a CLI but that's a separate repo, and it's called the `fly` CLI.
```suggestion
display_name: Concourse CI
```

File: topics/concourseCli/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+topic: concourseCli
```
Comment (deniseyu) [inline topics/concourseCli/index.md:3]:
```suggestion
topic: concourse-ci
```

File: topics/concourseCli/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+github_url: https://github.com/concourse/concourse
+logo: concourseCli.png
```
Comment (deniseyu) [inline topics/concourseCli/index.md:5]: This is a nitpick but would you mind renaming the PNG file accordingly?
```suggestion
logo: concourse-ci.png
```

**Comment by AshishVarshneyy** [comment]
If it's okay then add the **hacktoberfest-accepted** label while merging this PR.

**Comment by AshishVarshneyy** [comment]
If it's okay then add the **hacktoberfest-accepted** label while merging this PR.

**Comment by deniseyu** [comment]
Hey @AshishVarshneyy, this repo is not participating in Hacktoberfest so contributions here won't count as per the updated rules: https://hacktoberfest.digitalocean.com/details

**Comment by AshishVarshneyy** [comment]
okay no problem

**Comment by deniseyu** [comment]
We've made the decision not to opt this repository into Hacktoberfest. We appreciate the contribution, and if you're looking for Hacktoberfest projects I'd recommend you filter for projects that have opted in using the `hacktoberfest` label!

**Comment by AshishVarshneyy** [comment]
i do not have any problem u can merge it as it is.

**Review by damaneice — APPROVED** [review]

**Pull request merged**